### PR TITLE
README update

### PR DESCRIPTION
**Pull request by bboyatwork**
Addressing #68 Clarifying Clutter install path and links.

### CODE REVIEW

**Comment by zippy** [comment]
Hey @science-girl could you please give my readme edits a once over and then approve?

File: README.md [hunk 1]
```diff
@@ -3,47 +3,86 @@
...
+
+``` shell
+cd clutter
+hcdev web
+```
+and point your browser at `http://localhost:4141` to access the UI.
+
+**--or--**,
+
+assuming that you've already setup you Holochain environment with [`hcadmin init`](https://developer.holochain.org/Command_Line_Tools#hcadmin_init) you can join and run Clutter with:
```
Comment (science-girl) [inline README.md]: you -> your

File: README.md [hunk 2]
```diff
@@ -3,47 +3,86 @@
...
+hcadmin join . clutter
+hcd clutter
+```
+and point your browser at `http://localhost:3141` to access the UI.
+
+## Installation: for Developers
+
+**Prerequiste:** [Install Holochain](http://developer.holochain.org/Install_Holochain) on your machine and make sure you do the step to set the $GOPATH.
+
+**Dependencies:** If you want to work on the Clutter UI, you will need `nodejs` (https://nodejs.org/en/) (LTS) installed, with `npm` or `yarn` (https://yarnpkg.com/lang/en/docs/install) to be able to build it from the `ui-src` to the `ui` directory:
```
Comment (science-girl) [inline README.md]: "...to be able to build and copy it from the 'ui-src'..."

File: README.md [hunk 3]
```diff
@@ -3,47 +3,86 @@
...
+
+The following commands will clone the latest build of clutter to your machine (you may want to use your own fork instead of our repo) and then you will either install npm or yarn to build the UI for the app.
 ```
 git clone https://github.com/Holochain/clutter.git
 cd clutter/ui-src
 npm install # (or yarn install)
 npm run build # (or yarn build)
 cd ..
```
Comment (science-girl) [inline README.md]: After ```npm run build```, ```npm start``` to configure and start the React UI.

File: README.md [hunk 4]
```diff
@@ -3,47 +3,86 @@
...
+
+Make two copies of your your clutter folder one called `clutter1` and the other `clutter2`. Both folders will need to have a dna folder and a ui folder in each.
 
-Firstly run the bootstrap server which will let each instance of Clutter know about its peers.  The ```bs```  command is part of the Holochain install.  If it doesn't work you probably need to set the $GO_PATH variable. (Soon we won't need this step)
+Firstly run the bootstrap server which will let each instance of Clutter know about its peers.  The ```bs```  command is part of the Holochain install. If it doesn't work you probably need to set the $GO_PATH variable. (Soon we won't need this step)
 ```
   bs
 ```
-You will get a response like
+You will get a response likev
```
Comment (science-girl) [inline README.md]: likev -> like:

File: README.md [hunk 5]
```diff
@@ -132,7 +171,7 @@ if running all in one terminal you will need to kill the processes between resta
 
 ## What the Automated build does
 
-When a branch is pushed to Github Travis runs a build.  The build does the following:
+When a branch is pushed to Github Travis runs a build. The build does the following:
 1. Installs docker-compose
 2. Runs docker-compose up -d which spins up a bootstrap server and 3 instances of clutter
 3. Install the cypress dependencies
```
Comment (science-girl) [inline README.md]: For the 3rd point: Install -> Installs

**Comment by zippy** [comment]
done.

**Pull request merged**